app/helpers/filter-replace.js

```diff
@@ -7,6 +7,19 @@ var filters = {
     'Research Papers in Economics': 'RePEc'
 };
 
+/**
+ * @module ember-preprints
+ * @submodule helpers
+ */
+
+/**
+ * filterReplace helper. Replaces long provider names without messing with search filter logic
+ *
+ * @class filterReplace
+ * @param {String} filter Filter
```

Comment:
I am not sure. I had originally used `method` here, but it wasn't showing up in the docs.